library/ceph_volume.py

```diff
@@ -636,18 +636,22 @@ def run_module():
         # Zap the OSD
         skip = []
         for device_type in ['journal','data', 'db', 'wal']:
+            # 1/ if we passed vg/lv
```

Comment:
should be reworked but I don't really want to introduce big changes for a downstream zstream release